tracetools_trace/tracetools_trace/tools/lttng_impl.py

```diff
@@ -190,15 +193,27 @@ def setup(
         the usual page size)
     :param subbuffer_size_kernel: the size of the subbuffers for kernel events (defaults to 32
         times the usual page size, since there can be way more kernel events than UST events)
+    :param live_mode: whether to create a live session
+    :param live_timer_interval: the time interval at which the data should be flushed from the
+        buffer and sent to the LTTng relay. This is in microseconds.
+        Used only if live_mode is `True`.
+    :param live_tracing_url_origin: the URL to which the tracing output will be sent.
+        Used only if live_mode is `True`.
     :return: the full path to the trace directory, or `None` if initialization failed
     """
     # Validate parameters
     if not session_name:
         raise RuntimeError('empty session name')
     # Resolve full tracing directory path
-    # TODO(christophebedard): do we need to join the base_path with session_name for a live session?
+    # TODO(christophebedard): do we need to join the
+    #   base_path with session_name for a live session?
     #   We need to return a path, so maybe format it like:
     #   "net://localhost/host/$hostname/$session_name"
+    live_tracing_url = (
+        live_tracing_url_origin + '/' +
+        'host/' + socket.gethostname() + '/' +
+        session_name
+    )
```

Comment:
I actually don't yet fully understand what this URL represents. I think this works if you process the live trace from the same computer, but what should this URL be if you want to process the trace data from another computer and what URL should the other computer use? I think this is a pretty important use-case.

Would you be able to look into what the URL represents (if you haven't already)? If you have two computers on a local network that you can try this with, that would also help us understand. You could use the `bt2_lttng_live.py` script from #149 to receive trace data.